### DISCUSSION TITLE
missing -wait?

### DISCUSSION THREAD

**Discussion by thejck** [nginx-proxy/acme-companion#1105] [Q&A]
https://github.com/nginx-proxy/acme-companion/blob/b22b6ef76067949456cb6ae5979f194005c7aa4b/docs/Docker-Compose.md?plain=1#L78

should this line include a -wait before "5s:30s"

 command: -notify-sighup nginx-proxy -watch -wait 5s:30s /etc/docker-gen/templates/nginx.tmpl /etc/nginx/conf.d/default.conf

**Comment by buchdag** [comment]
@thejck good catch, thank you 👍 , I'll edit the doc.
